src/components/pager/__internal__/pager-navigation.spec.js

```diff
@@ -41,7 +40,7 @@ describe("Pager Navigation", () => {
     assertStyleMatch(
       {
         display: "flex",
-        flex: "1 1 auto",
+        flex: "1 1 100%",
```

Comment:
Why've you changed this flex-basis to 100%?

Reply:
Yeah good spot, had it set up like that when I was conditionally rendering the other flex children but I shifted it to conditionally render the children only of these sub-components so the spacing is maintained. Will revert it back to `auto`